versioneer.py

```diff
@@ -276,11 +276,7 @@
 
 """
 
-from __future__ import print_function
-try:
-    import configparser
-except ImportError:
-    import ConfigParser as configparser
+import configparser
```

Comment:
While these are valid updates, I'm disinclined to change versioneer except for functional fixes, to keep any diffs focused.

Reply:
Ok, what about the changes after the imports? They are deprecated in py3.6

Reply:
Fair enough. Let's go ahead with these changes, then.